### PR TITLE
LPS-193017 [PRM] Fixed Revenue Dashboard Showing incorrect data

### PR DESCRIPTION
**Pull request by Elvinho**

### CODE REVIEW

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/common/components/dashboard/utils/formatCurrency.ts [hunk 1]
```diff
@@ -5,15 +5,17 @@
 
 import {Liferay} from '../../../services/liferay';
 
-const formatCurrency = (value: any, currencyKey?: string) =>
-	new Intl.NumberFormat(Liferay.ThemeDisplay.getBCP47LanguageId(), {
+const formatCurrency = (value: number, currencyKey?: string) => {
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/common/components/dashboard/utils/formatCurrency.ts:8]:
```suggestion
const formatCurrency = (value: number, currencyKey?: string, precision?: string) => {
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/common/components/dashboard/utils/formatCurrency.ts [hunk 2]
```diff
@@ -5,15 +5,17 @@
...
+	return new Intl.NumberFormat(Liferay.ThemeDisplay.getBCP47LanguageId(), {
 		currency: currencyKey ? currencyKey : 'USD',
 		notation: 'compact',
 
 		// @ts-ignore
 
-		roundingPriority: 'morePrecision',
+		roundingPriority: 'lessPrecision',
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/common/components/dashboard/utils/formatCurrency.ts:15]:
```suggestion
		roundingPriority: precision || 'morePrecision',
```

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/common/components/dashboard/components/DonutChart/index.tsx [hunk 3]
```diff
@@ -76,7 +76,8 @@ const DonutChart = ({
 		return (
 			<>
 				<span className="text-nowrap">
-					{titleChart} <b>{valueChart}</b>
+					{titleChart}{' '}
+					<b>{formatCurrency(valueChart, dataCurrency)}</b>
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/common/components/dashboard/components/DonutChart/index.tsx:80]:
```suggestion
{titleChart} <b>{valueChart}</b>
```

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

**Comment by Elvinho** [comment]
ci:test:sf

**Comment by jairmedeiros** [comment]
ci:forward

**Pull request closed without merging**